README.md

```diff
@@ -190,6 +190,21 @@ end
 
 Scripttags are created in the same way as the Webhooks, with a background job which will create the required scripttags.
 
+If you want to use the application.js file as generated by the Rails asset pipeline, you can do so by moving the `ShopifyApp.configure`
```

Comment:
I agree -- I just wanted to get a demo app working with integration as a proof of concept so I was using `application.js`.